static/css/training.css

```diff
@@ -139,20 +81,77 @@ section.call-to-action .main-section .cta-image > img {
   text-decoration: none;
 }
 
+.mainsection {
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.second {
+  padding-top:4%;
+}
+
 h5 {
   font-size: 16px;
   font-weight: 500;
   line-height: 1.5em;
   margin-bottom: 2em;
 }
 
+.logo-certification .cta-image .cta-image-after img {
+  width:300px !important;
+}
+
 .white-text {
   color: #fff;
 }
 
 .padded {
-  padding-top: 100px;
-  padding-bottom: 100px;
+  padding-top: 3%;
+  padding-bottom: 4%;
+}
+
+.padded-bottom-banner {
+  padding-top:0%;
+  padding-bottom: 2%;
+}
+
+.half {
```

Comment:
This class breaks a little at larger screen sizes: 

<img width="793" alt="Screenshot 2020-05-11 at 10 37 01" src="https://user-images.githubusercontent.com/17999517/81592813-5da77880-9373-11ea-96c2-9633fc5d4536.png">

This is a multi-part problem to fix across multiple files in this PR.

**Part 1:***
Removing most of the padding seems to fix it: 
```
.half {
  width:50%;
  float:left;
  padding-right:10%;
  padding-top:4%;
}
```